target/linux/ramips/image/rt305x.mk

```diff
@@ -389,6 +392,27 @@ define Device/dlink_dir-320-b1
 endef
 TARGET_DEVICES += dlink_dir-320-b1
 
+define Device/dlink_dir-506l
+  SOC := rt5350
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DIR-506L
+  DEVICE_PACKAGES := jboot-tools kmod-usb2 kmod-spi-dev kmod-usb-serial \
+		kmod-usb-serial-option kmod-usb-net \
```

Comment:
usb-serial and usb-net can be dropped, as they will be pulled by the other packages